python/tests/test_parallel_flow.py

```
@@ -126,8 +126,8 @@ async def test_execute_triggered_branches_concurrently(self, setup):
         assert isinstance(process_c_results, list) and len(process_c_results) == 1, "'process_c' should be a list with 1 result"
         
         # Check that both branches completed
-        assert process_b_results[0] == {DEFAULT_ACTION: []}
-        assert process_c_results[0] == {DEFAULT_ACTION: []}
+        assert process_b_results[0] == None
```

Ruff (0.11.9)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)